### ISSUE TITLE
Incompatibility with newest @types/jest

### ISSUE THREAD

**Issue by Shadowstep33** [jest-community/jest-extended#234]
**Bug**
- `package` version: 0.11.2
- `node` version: 10.15.3
- `npm` (or `yarn`) version: 6.4.1

What you did: `npm i @types/jest@24.0.20` (latest)

What happened (please provide anything you think will help):

```
node_modules/@types/jest/index.d.ts:639:15 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

639     interface Matchers<R, T> {
                  ~~~~~~~~

node_modules/jest-extended/types/index.d.ts:5:13 - error TS2428: All declarations of 'Matchers' must have identical type parameters.

5   interface Matchers<R> {
              ~~~~~~~~
```

Reproduction repository (if possible):
run the following two commands:

```
npm i @types/jest@24.0.20` jest-extended@0.11.2`
npm run build
```

Description:
The definitions of the Matchers interface are out of sync

Possible solution:
Fixing the definition of Matchers should fix this

**Comment by Shadowstep33** [comment]
PR raised that should fix it here https://github.com/jest-community/jest-extended/pull/235

**Comment by scottbrassfield** [comment]
Hi, just wondering if there's any progress on getting this resolved. We want to add `jest-extended` to our project but will probably hold off until this is addressed, so we can keep up to date with jest types and not introduce workarounds. Thanks!

**Comment by NonPolynomial** [comment]
Meanwhile I use this `global.d.ts` as a workaround...

```ts
interface anonFn {
  (...args: any[]): any;
}

interface testSuite extends anonFn {
  skip: anonFn;
}

declare const describe: testSuite;
declare const it: testSuite;
declare const expect: anonFn;
```

**Comment by MikeMitterer** [comment]
Any news on this? Can't compile my package since 2 month because of this issue... Please publish a new version. Thanks

**Comment by mattphillips** [comment]
Fixed released in https://www.npmjs.com/package/jest-extended/v/0.11.3

**Comment by plaa** [comment]
Note that the fix is *not* in `jest-extended@0.11.3` but instead in `@types/jest`.
